sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -216,7 +216,7 @@ public async Task GetTablesReturnsTablesWithAndWithoutPagination(int? pageCount)
 
                 // Get the table list.
                 var remainingItems = createdTables.Count;
-                await foreach (var page in service.QueryAsync(/*maxPerPage: pageCount*/).AsPages(pageSizeHint: pageCount))
+                await foreach (var page in service.QueryAsync( /*maxPerPage: pageCount*/).AsPages(pageSizeHint: pageCount))
```

Comment:
```suggestion
                await foreach (var page in service.QueryAsync( /*maxPerPage: pageCount*/ ).AsPages(pageSizeHint: pageCount))
```

The lack of symmetry was setting off my OCD.  :smile: